README.md

```diff
@@ -1 +1,106 @@
-# webpacktodolist
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+<img src="./microverse.png" alt="logo" width="140"  height="auto" />
+<br/>
+
+<h3><b>Microverse README Template</b></h3>
+
+</div>
+
+
+# 📗 Table of Contents
+
+  - [🚀 Live Demo](https://pierrebarba.github.io/finishProjectCapton/index.html)
+- [💻 Getting Started](https://pierrebarba.github.io/finishProjectCapton/index.html)
+  - [Setup](setup)
+  - [Prerequisites](https://pierrebarba.github.io/finishProjectCapton/index.html)
+  - [Install](install)
+  - [Usage](html,javascript,css)
+- [👥 Authors](https://github.com/PierreBarba)
+- [🔭 Future Features](https://pierrebarba.github.io/finishProjectCapton/index.html)
+- [🤝 Contributing](https://github.com/cilfonegabriel)
+- [⭐️ Show your support](support)
+- [🙏 Acknowledgements](https://github.com/elafildecolor,https://github.com/luigirazum)
+- [📝 License](MIT)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Capton Project] <a name="Todo List"></a>
+
+
+**[Personal Portfolio Pierre]** is a...
+
+## 🛠 CSS,HTML,JAVASCRIPT <a name="javascript,html,css"></a>
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[dawesome]**
+- **[webpack]**
+- **[functional]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+
+
+<!-- AUTHORS -->
+
+## 👥 Pierre barba <a name="Pierre Barba"></a>
+
+👤 **Pierre barba arredondo**
+
+- GitHub: [@githubhandle](https://github.com/PierreBarba)
+- Twitter: [@twitterhandle](https://twitter.com/pierreanbar)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[practice more javascript]**
+- [ ] **[practice more abute localstorage]**
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- Microverse 
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+
```

Comment:
- [OPTIONAL] I suggest removing this `NOTE ...` recommendation from your README file 👍

Reply:
i will do that thanks